src/workingtitle-vcockpits-instruments-navsystems-g3000/html_ui/WorkingTitle/Pages/VCockpit/Instruments/NavSystems/AS3000/AS3000_Common.js

```diff
@@ -3,7 +3,7 @@ class AS3000_MapElement extends MapInstrumentElement {
         super();
 		this.simVarNameID = _simVarNameID;
 		
-		this.lastOrientation = 0;
+		this.orientation = 0;
```

Comment:
I like this overall, mainly have that thought on compatibility in MapInstrument.js.    If we can get that figured out I'd be in favor of merging this, although I would suggest perhaps we integrate the G1000 and GX with this branch first if whe don't keep compatiblity in MapInstrument.